content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -207,8 +216,20 @@ Some example values of `matchImages` patterns are:
 * 如果 imageMatch 包含端口，则该端口也必须在镜像中匹配。
 
 `matchImages` 模式的一些示例值：
+
 * `123456789.dkr.ecr.us-east-1.amazonaws.com`
 * `*.azurecr.io`
 * `gcr.io`
 * `*.*.registry.io`
 * `foo.registry.io:8080/path`
+
+## {{% heading "whatsnext" %}}
+
+<!--
+* Read the details about `CredentialProviderConfig` in the
+  [kubelet configuration API (v1alpha1) reference](/docs/reference/config-api/kubelet-config.v1alpha1/).
+* Read the [kubelet credential provider API reference (v1alpha1)](/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/).
+-->
+* 阅读 [kubelet 配置 API (v1alpha1) 参考](/zh/docs/reference/config-api/kubelet-config.v1alpha1/)中有关 `CredentialProviderConfig` 的详细信息。
+* 阅读 [kubelet 凭据提供程序 API 参考 (v1alpha1)](/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)。
```

Comment:
```suggestion
* 阅读 [kubelet 凭据提供程序 API 参考 (v1alpha1)](/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1/)。
```

Reply:
Thanks, but the zh page for kubelet-credentialprovider.v1alpha1.md does not exist at present, so I recommend linking it to the en page for now. WDYT?

Reply:
At this stage, there are many dangling links to be fixed (both in the En directory and the Zh directory). We can worry about that later.

Reply:
Understood, thanks.